### ISSUE TITLE
`docker build` own image from the reaction directory failed 

### ISSUE THREAD

**Issue by weitony111** [reactioncommerce/reaction#1674]
For bugs, please fill out the three following sections completely

### Expected behavior
I expect that the build should succeed and there is `STDOUT` output as well as `STDERR` output if it is failed.
### Actual Behavior
`docker build` stopped with exit code of 1. Screen outputs are as follows and it doesn't have `STDERR` but only `STDOUT`.
>[-] Running Reaction plugin loader...

> [-] Running npm install...

> npm WARN deprecated node-uuid@1.4.7: use uuid module instead

> fibers@1.0.15 install /opt/reaction/src/node_modules/fibers
> node build.js || nodejs build.js

> The command '/bin/sh -c bash $BUILD_SCRIPTS_DIR/install-meteor.sh &&             bash $BUILD_SCRIPTS_DIR/build-meteor.sh &&             bash $BUILD_SCRIPTS_DIR/post-build-cleanup.sh' returned a non-zero code: 1

### Steps to Reproduce the Behavior
1. Create a new Amazon EC2 instance
2. `SSH` to it
3. `docker build https://github.com/reactioncommerce/reaction.git`

**Comment by jshimko** [comment]
A few things...

First, if you're just going to use the master branch to build an image, you should skip the build step and use the [official image on Docker Hub](https://hub.docker.com/r/reactioncommerce/reaction/).  It's 100% identical to the master branch because it automatically builds every time we merge to master.

Second, if you need a custom build for some reason, you should be building the base image at the same time you build the app image.  There's a [build script](https://github.com/reactioncommerce/reaction/blob/master/.reaction/docker/build.sh) that does that, so run this instead:

```sh
.reaction/docker/build.sh
```

**Comment by weitony111** [comment]
Hi @jshimko 

Thanks for your reply. 

The reason why I didn't use the official image on Docker Hub is that I want to figure out whether port 3000 or port 80 is exposed by the container. Since in `.reaction/docker/base.dockerfile`, 3000 is exposed but in the description of the official image on Docker Hub port 80 is used.

Regarding my previous issue, I have also used the build script to build the image; however, same issue happened.

After investigation, it is found that it is because in Amazon EC2 ubuntu instance, there is no Swap space in it, which may cause `npm install` failed. I followed [this](https://www.digitalocean.com/community/tutorials/how-to-add-swap-on-ubuntu-14-04) to add Swap on it and the image is successfully built.

Thank you.
